packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -95,7 +94,7 @@ export const getTableStyles = (theme: GrafanaTheme2) => {
     cellHeight,
     buildCellContainerStyle,
     cellPadding,
-    cellHeightInner: bodyFontSize * lineHeight,
+    cellHeightInner: cellHeight - cellPadding * 2,
```

Comment:
One thing I see we need to fix (either in this PR or direct followup) is to break the link between cell height and header cell height, I think just because you want more height for the cells the header row does not need to increase (looks a bit strang as well with the header cell text not being centered. And ideally I want to reduce the header height (it could be a  bit shorter than even sm data rows)

Reply:
Just pushed a fix for this that makes header and footer always be 28px which I think is a better height for those rows